utest.h

```diff
@@ -334,7 +334,7 @@ static UTEST_INLINE utest_int64_t utest_ns(void) {
   QueryPerformanceCounter(&counter);
   QueryPerformanceFrequency(&frequency);
   return UTEST_CAST(utest_int64_t,
-                    (counter.QuadPart * 1000000000) / frequency.QuadPart);
+                    (counter.QuadPart / frequency.QuadPart * 1000000000));
```

Comment:
Doesn't this mean you will lose all timing for short running tests? The divide by frequency will round to 0?

Reply:
> Doesn't this mean you will lose all timing for short running tests? The divide by frequency will round to 0?
Yes, but no undefined behaviour, maybe use floats?

Reply:
Hello, I've found the same issue. I've fixed it in the same way sokol_time computes the ns https://github.com/floooh/sokol/blob/e85b58aaf2e1c7603a0bd0eea7e99a250948299b/sokol_time.h#L204

Here is my commit:
https://github.com/tanoxyz/utest.h/commit/4e300d6c5cbc64de6c112a73c5fa94fb3de2cc92

Reply:
Thank you!